src/data/api/datatyper/apiGetOptions.ts

```diff
@@ -1,7 +1,9 @@
 export const GEToptions = {
     method: 'GET',
     headers: {
+        'Nav-Consumer-Id': 'veilarbdetaljerfs',
         Accept: 'application/json',
-        'Content-Type': 'application/json;charset=UTF-8'
+        'Content-Type': 'application/json;charset=UTF-8',
+        credentials: 'include'
```

Comment:
Før vi spesifiserer `credentials: 'include'` (i stedet for f.eks. `credentials: 'same-origin'`) så må vi være sikre på at dette gjøre det vi ønsker. Ref. [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#sending_a_request_with_credentials_included), når vi bruker `credentials: 'include'` så tillater vi at nettleseren sender ved cookies, mm. (aka. credentials), til andre sider utenfor samme origin som frontendkoden vår kjører på.

Reply:
Den er notert! Vi fjernet hvertfall credentials for nå:D